src/color_button.rs

```diff
@@ -34,9 +27,9 @@ impl ColorButton {
     }
 
     pub fn get_rgba(&self) -> gdk::RGBA {
-        unsafe { 
-            let mut rgba = mem::uninitialized();
-            ffi::gtk_color_button_get_rgba(self.to_glib_none().0, &mut rgba);
+        unsafe {
```

Comment:
This function and `set_rgba` also not needed as it implemented by `ColorChooserExt`
I don't know why rust now don't nags about multiple cases even on use.